tensorflow/python/kernel_tests/shape_ops_test.py

```diff
@@ -48,6 +48,15 @@ def _compareShape(self, x, use_gpu=False):
     self.assertAllEqual(np_ans, result)
     self.assertShapeEqual(np_ans, tf_ans)
 
+  def _compareShapeSparse(self, x_np, use_gpu=False):
+    np_ans = np.array(np.shape(x_np))
+    x_tf, nnz = _sparsify(x_np)
```

Comment:
Nit: for unused arguments, we prepend the "unused" prefix to pass the linter.  Could you rename "nnz" to "unused_nnz"?  (Sorry for not catching this last time.)